app/post-contribution/route.js

```diff
@@ -17,7 +17,9 @@ export default Route.extend(AuthenticatedRouteMixin, {
   model({ date = moment().format('YYYY-MM-DD') }) {
     let user = this.get('currentUser.user')
 
-    return this.store.createRecord('contribution', { user, date })
+    let challenge = this.store.findRecord('challenge', 'current')
```

Comment:
Right. How can I make sure the relation is resolved for the contribution model that is loaded in the edit-contribution route?